helloworld/src/main/java/com/helloworld/helloworld/HelloWorldController.java

```diff
@@ -2,10 +2,20 @@
 
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
+import java.util.List;
 
 @RestController
 public class HelloWorldController {
     @GetMapping("/helloworld")
     public String helloworld(){return "Hello World!";}
 
-}
+    @GetMapping("/worldgreeting")
+    public List<Greeting> worldgreetinglist(){
```

Comment:
メソッド名はキャメルケースでお願いします。
```suggestion
    public List<Greeting> worldGreetingList(){
```